### ISSUE TITLE
Not working in Windows

### ISSUE THREAD

**Issue by bayasdev** [carlini/printf-tac-toe#3]
Windows 10 LTSC 2019
TDM-GCC 9.2

![Captura de pantalla (6)](https://user-images.githubusercontent.com/49411169/83978603-60dc4880-a8ce-11ea-9dc2-b55be433f615.png)

**Comment by LinoBigatti** [comment]
I think this relies on POSIX printf.

**Comment by carlini** [comment]
Yeah, I tested this on most variants of Linux I could come across and OSX. It wouldn't surprise me if it wasn't compatible with Windows.

From your output, it looks like it's the fact that I use `%N$` to reduce the number of necessary arguments to printf. This is something that is also nonstandard. The `printf` implementation might decide to just treat this as a raw string to print. There might be some way to get this behavior on Windows---but I don't know. If not, one "fix" would be to duplicate the arguments to printf so that it can just use the standard %s and read the next argument.

(It's also possible that this won't work for another reason: the macro-expanded format string is ~100,000 bytes long, which significantly exceeds the maximum that the C standard says compilers must allow. I think that's 4095 bytes for c99.)

**Comment by ret394** [comment]
this code is awesome. only thing left is to mention requirements explicitly in readme.

thanks. have a nice day. keep up the great work. 👏

[ret394](https://github.com/ret394)
